sql/src/test/java/io/crate/integrationtests/DDLShrinkIntegrationTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.integrationtests;
+
+import io.crate.testing.UseRandomizedSchema;
+import org.elasticsearch.cluster.service.ClusterService;
+import org.elasticsearch.test.ESIntegTestCase;
+import org.junit.Test;
+
+import static com.carrotsearch.randomizedtesting.RandomizedTest.$;
+import static org.hamcrest.core.Is.is;
+
+@ESIntegTestCase.ClusterScope(minNumDataNodes = 2)
+@UseRandomizedSchema()
```

Comment:
Both annotations can be removed as they are defining default values.

Reply:
I know, just wanted it to be explicit/clear - anyway, I'll remove them

Reply:
actually, isn't the default for min nodes `DEFAULT_MIN_NUM_DATA_NODES` = 1 ?

Reply:
ah right, seems to have been changed. but are your tests requiring min 2 nodes? if not please remove to always rely on defaults.

Reply:
no, they are not required  - just wanted to have them covered. 
I guess defaults are fine in this case.